salt/modules/win_service.py

```diff
@@ -484,6 +625,15 @@ def start(name, timeout=90):
 
             .. versionadded:: 2017.7.9,2018.3.4
 
+        with_deps (bool):
+            If enabled start the given service and the services the current
+            service depends on.
+
+        with_parents (bool):
```

Comment:
This name is confusing... sounds more like Children... but not quite. Maybe use `with_depends_on`, though I'm not sold on that either.

Reply:
Sure, I can do that. The fun fact here is that there was a discussion about that name already and now guess who suggested to use "parents" ;)